src/test/regress/pg_regress_multi.pl

```diff
@@ -920,56 +920,72 @@ END
         system(catfile($bindir, "psql"),
             ('-X', '-h', $host, '-p', $port, '-U', $user, "-d", "postgres",
                 '-c', "CREATE DATABASE regression;")) == 0
-            or die "Could not create regression database on worker";
+            or die "Could not create regression database on worker port $port.";
+        print "Created database regression on worker port $port. \n";
+        system(catfile($bindir, "psql"),
```

Comment:
I think showing alone could be fragile. What if this behavior changes? We cannot automatically detect.

Instead, can we maybe change the script such that if `shared_preload_libraries` does not include `citus`, we go to `die` instead of just printing it?